utils/config.py

```diff
@@ -0,0 +1,251 @@
+"""
+BICEP Pipeline Configuration
+
+Centralized configuration for all file paths, directories, and settings used throughout
+the BICEP pipeline. This ensures consistent path handling across all modules.
+"""
+
+from pathlib import Path
+import os
+from loguru import logger
+
+# ============= BASE DIRECTORIES =============
+
+# Root directory of the BICEP project (utils/../ = project root)
+BICEP_ROOT = Path(__file__).parent.parent
+ROOT = BICEP_ROOT  # Keep Tim's variable name for compatibility
+
+# Data directories
+DATA_ROOT = BICEP_ROOT / 'data'
+DATA_DIR = DATA_ROOT  # Keep Tim's variable name for compatibility
+RAW_INPUTS_PATH = DATA_ROOT / 'raw_inputs'
+PARSED_INPUTS_PATH = DATA_ROOT / 'parsed_inputs'
+REQUIRED_INPUT_PATH = DATA_ROOT / 'required_input'
+
+# ============= DATA LOCATION SETTING =============
+
+DATA_LOCATION = 'LOCAL'
+try:
+    assert DATA_LOCATION in ('LOCAL', 'PNNL Database')
+except AssertionError as error:
+    logger.error(f'DATA_LOCATION {DATA_LOCATION} is not valid. Must be in ["LOCAL", "PNNL Database"]')
+    logger.error(error)
+    raise AssertionError
+
```

Comment:
Let's add logic here to run check for the local files if it is in "LOCAL" model. 

```suggestion
if DATA_LOCATION == 'LOCAL':
    ensure_data_assets()
```